### PR TITLE
README improvements

### PR DESCRIPTION
**Pull request by nearlySplat**
### Changes:

- add alt text to images
- remove `</br>` fluff 
- improve manual installation instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,49 @@
 <center><img src='resources/icons/logoLarge.svg' height='100px' style='border-radius: 10px;'/></center>
 
-## LiveBot is specifically designed for bots; it cannot be used for user accounts. 
+> LiveBot is specifically designed for bots; it cannot be used for user accounts. Furthermore, using user account tokens not in an official client is aginst the Discord Community Guidelines.
```
Comment (SebOuellette) [inline README.md:3]: Unnecessary

File: README.md [hunk 2]
```diff
@@ -1,45 +1,49 @@
 <center><img src='resources/icons/logoLarge.svg' height='100px' style='border-radius: 10px;'/></center>
 
-## LiveBot is specifically designed for bots; it cannot be used for user accounts. 
+> LiveBot is specifically designed for bots; it cannot be used for user accounts. Furthermore, using user account tokens not in an official client is aginst the Discord Community Guidelines.
+
 An app that allows you to be inside a bot!
 
-At the moment, LiveBot is still in alpha. Installation instructions can be found below.
+At the moment, LiveBot is still in alpha development. Installation instructions can be found below.
```
Comment (SebOuellette) [inline README.md:7]: Implied

File: README.md [hunk 3]
```diff
@@ -1,45 +1,49 @@
...
-At the moment, LiveBot is still in alpha. Installation instructions can be found below.
+At the moment, LiveBot is still in alpha development. Installation instructions can be found below.
 
 ## Discord
 If you would like to get the latest information on the project, talk with the developers, post suggestions, get help, etc., join the official LiveBot [Discord server!](https://discord.gg/NG4rgqSgzx)
 
 ## What is this?
-<img src='https://i.imgur.com/m0HvVmn.png' height='300px' style='border-radius: 10px;'/>
-<br>
+![preview](https://i.imgur.com/m0HvVmn.png)
```
Comment (SebOuellette) [inline README.md:13]: I'd like to keep my border radius

File: README.md [hunk 4]
```diff
@@ -1,45 +1,49 @@
...
+
 LiveBot is a program built with electron and discord.js, that will allow you to have discord controlling a bot. You are able to view messages in channels, send messages, interact with people, and see what is going on in the servers your bot is in. LiveBot can only be used on bot accounts, if you find a way to use it on user accounts we're in no way liable for anything that might happen.
-</br>
+
 LiveBot had been built so that the style and flow is similar to discord, and while it is not exact, it is pretty close. it is also built from scratch, using no frameworks, just straight html, css, and javascript. Due to this, and having a very small developer team, additions may take a while to be added. The point is to built from scratch, we know very well that it's not the fastest way to make something like this. However, LiveBot is slowly being filled with cool exclusive features like the pull-up user settings menu, the soon to come embed builder, native support for scripts, and more!
 
 ## How to install
-### Manual (Necessary for Mac and Linux - Works on Windows too)
+### Manual (Necessary for Mac and Linux - works on Windows too)
```
Comment (SebOuellette) [inline README.md:20]: Would prefer this to be capitalized

File: README.md [hunk 5]
```diff
@@ -1,45 +1,49 @@
...
 **Before following any installation steps**, make sure you [install Node.js](https://nodejs.org/en/).
-To install LiveBot, just download as a zip file, which you can do by pressing the green button above that has `"Code"` written inside. Once you have downloaded and extracted the zip file, go into the directory with a command prompt or shell window. In the shell window, enter the command `npm install`. Make sure that the terminal window is running as administrator. Once this process is finished - it may take some time - type `npm start`. 
+You can install LiveBot in many ways such as Git, but the easiest way is to just download as a compressed (zip) file, which you can do by pressing the green button above that has `Code` written on it. Once you have downloaded and extracted the zip file, go into the directory with a command prompt or shell window. In the shell window, enter the command `npm install` to install the needed dependencies. Make sure that the terminal window is running as Administrator (or root, for Linux people). <!-- is this necessary? --> Once this process is finished — it may take some time — type `npm start`.
```
Comment (SebOuellette) [inline README.md:22]: Not a bad change, gives more information.

File: README.md [hunk 6]
```diff
@@ -1,45 +1,49 @@
...
+You can install LiveBot in many ways such as Git, but the easiest way is to just download as a compressed (zip) file, which you can do by pressing the green button above that has `Code` written on it. Once you have downloaded and extracted the zip file, go into the directory with a command prompt or shell window. In the shell window, enter the command `npm install` to install the needed dependencies. Make sure that the terminal window is running as Administrator (or root, for Linux people). <!-- is this necessary? --> Once this process is finished — it may take some time — type `npm start`. 
 
-### Easy (Windows Only)
-If you would like to just download and open an exe file, find the releases on the right side of this page, and download the latest release. You will need to download the source code and the exe file, then place the executable in the source code folder. 
+### Easy (Windows only)
+If you would like to just download and open an executable file, find the releases on the right side of this page, and download the latest release. You will need to download the source code and the executable file, then place the executable in the source code folder.
```
Comment (SebOuellette) [inline README.md:25]: Would rather it says exe here

File: README.md [hunk 7]
```diff
@@ -1,45 +1,49 @@
...
-This box is how you log into a profile after already logging in. To log into a token by default, put it into this box then press the save button. Otherwise if you are just logging into a token just for a look, input the token into the box then press `enter`.<br>
+This box is how you log into a profile after already logging in. To log into a token by default, put it into this box then press the save button. Otherwise if you are just logging into a token just for a look, input the token into the box then press `enter`.
+
 The token changer used to be at the top of the main screen, but has moved to the user options menu.
-<img src='https://i.imgur.com/1rnEpQD.png' style='border-radius: 10px;'>
-</br>
 
-### Barry!
+![token changer](https://i.imgur.com/1rnEpQD.png)
+
```
Comment (SebOuellette) [inline README.md:34]: Again, would like to keep my border radius. Also this says Token changer

File: README.md [hunk 8]
```diff
@@ -1,45 +1,49 @@
...
+### Barry
 Barry is the LiveBot version of discord's Clyde. Except Barry has some fun commands! Just use the prefix `/` and to see the commands write `/help`. Only you can see what barry says, so don't worry about interfering with any conversations. <br>
-<img src='https://i.imgur.com/PGInuit.png' style='border-radius: 10px;'>
+![Barry](https://i.imgur.com/PGInuit.png)
```
Comment (SebOuellette) [inline README.md:37]: border radius

File: README.md [hunk 9]
```diff
@@ -1,45 +1,49 @@
...
 
-### Profile card.
+### Profile card
 The profile card is able to tell you the username, the avatar image, the discriminator, and something discord does not have. The profile card will tell you if the account you are signed into is a bot or a user. This can be handy if you are not quite sure. As you can no longer login with a user account it's just legacy code and for the looks. This card also has the pull up menu on the right side, so you can get to all the settings. <br>
-<img src='https://i.imgur.com/79NacEx.png' style='border-radius: 10px;'>
 
+![profile card](https://i.imgur.com/79NacEx.png)
```
Comment (SebOuellette) [inline README.md:42]: border radius

File: README.md [hunk 10]
```diff
@@ -1,45 +1,49 @@
...
 
 ### Generate Invite
-LiveBot gives you the ability to generate an invite quickly from within the app. You won't have to go to the discord developers page! It can be found in the user settings page in the pull up menu. Just toggle all the permissions you'd like to give the invite (by default, everything recommended to use LiveBot's features is enabled). 
-<br>
-<img src='https://i.imgur.com/ggkyQtn.png' style='border-radius: 10px;'>
+
+LiveBot gives you the ability to generate an invite quickly from within the app, instead of having to go to your bot's application page in the Discord Developer portal. It can be found in the user settings page in the pull up menu. Just toggle all the permissions you'd like to give the invite (by default, everything recommended to use LiveBot's features is enabled). 
+
+![Invite Generator](https://i.imgur.com/ggkyQtn.png)
```
Comment (SebOuellette) [inline README.md:48]: border radius

**Review by SebOuellette — COMMENTED** [review]
I've finished the review so you can take a look at it. There's more that I'm not a fan of than what I am a fan of. 

I would also like to keep the <br> so that third party github clients are more likely to format the markdown correctly. I added them because my preview in vscode didn't display properly, so I'd assume other things can as well.

File: README.md (hunk 7 above)
Comment (nearlySplat) [reply README.md:34]: it's actually the token changer image, just Git displayed it below Barry in the diff

Comment (SebOuellette) [reply README.md:34]: Ah

**Comment by nearlySplat** [comment]
The code review changes have been applied! Thanks for the interest in my humble README improvements >w<

**Review by SebOuellette — APPROVED** [review]
Nice enhancement, might make some more changes myself later on so the wording isn't so weird haha

**Pull request merged**